pkg/debug/scripts/qemu-affinities.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#
+# Output affinities for each QEMU thread
+#
+
+# We need that for proper GNU version of the 'ps' tool
+apk add procps > /dev/null
```

Comment:
I think script should fail here (like add "... || exit 1" or do it better with an error message) if procps cannot be installed (because it's essential for this script). Also, isn't safe to do a "apk update" before the apk add... ? I'm wondering when an EVE version get too old for the apk repos and someone try to run the script.... BTW, apk will depend on network connection, so it shouldn't be too difficult to get an error here...

Reply:
You can also check if the package is installed with something like this:
```shell
apk list -I | grep ^procps
```